src/classroom.rb

```diff
@@ -0,0 +1,17 @@
+class Classrom
+  attr_accessor :label, :students
+
```

Comment:
- [ ] Kindly ensure that the `students `object can only retrieve the value of instance of the variable. Getters method only provide read access to the attributes of an object. ℹ️
```suggestion
  attr_reader :students
```